### ISSUE TITLE
[Request] Crag gen option

### ISSUE THREAD

**Issue by vagaprime** [Team-RTG/Realistic-Terrain-Generation#647]
Would be possible to add a config option to allow the crags to gen as they did before snapshot 3?

**Comment by whichonespink44** [comment]
Not sure we want to be putting in config options for terrain at this stage.

Can you post a screenshot of what they used to look like? Are there any specific features from the old version that we could maybe integrate into the new one to make it work for you?

**Comment by Zeno410** [comment]
I assume you mean the spiky crags, vagaprime? That were your idea, pink! lol. We'll probably add a biome modification system at some point, where outside mods can sub in their biomes, rather than have a blizzard of config options.

**Comment by vagaprime** [comment]
Ya first time I found a crag biome I loved the way they looked and thought that would make for an epic dwarf city. updated the pack I was playing and they updated RTG and I say the spiky crags and had a hard time being able to visualize being able to build anything. here is a picture of the old crag gen you had before snapshot 3

http://imgur.com/M6rxxEK

render distance set out to 16, anything more and my computer crashes

**Comment by topisani** [comment]
@Zeno410 in v2 outside mods can do just that - add their own realistic versions of biomes. And then in the config you can choose which one you want to go with

**Comment by whichonespink44** [comment]
Not sure what we should do about this one... we made the current version of the Crag based on previous user feedback, so if we change it back, it's only going to upset the others.

I think maybe we run with the current version unless more people come forward and suggest an alternative version.

So @vagaprime, if you want the old Crag back, you're gonna have rally some supporters and maybe get a discussion going in the forums. :wink: 

In the meantime, I think we'll stick with what we've got.
